custom-session-guide.md

```diff
@@ -1,67 +1,102 @@
 # Custom Session Configuration Guide
 
-This guide explains how to configure a custom session handler for the `ChatAiWidget` component. This can be useful if you need to set up authentication with your own API.
+This guide provides detailed instructions on configuring a custom session handler for the `ChatAiWidget` component, particularly useful for integrating authentication with your own API.
 
-### Step-by-Step Guide
+## Prerequisites
 
-#### Define the Session Configuration Function
-We recommend to memoize the configure function with `useCallback`. This ensures that the function is not recreated on every render.
-```jsx
-const issueSessionToken = async (userId, ...) => {
-  // Build your own API request handler here
+Before you can manage sessions independently, you must obtain session tokens using the Sendbird Platform API. For detailed instructions on preparing to use the Platform API, please refer to the [official documentation](https://sendbird.com/docs/chat/platform-api/v3/prepare-to-use-api).
+
+## Step-by-Step Guide
+
+### Defining a Function to Issue Session Tokens
+
+Define a function to issue session tokens. The following TypeScript code is an example; in practice, you should issue session tokens from your server. For more information, please see the [Sendbird documentation](https://sendbird.com/docs/chat/platform-api/v3/user/managing-session-tokens/issue-a-session-token).
+
+```ts
+const APP_ID = "YOUR_APP_ID";
+const API_TOKEN = "YOUR_API_TOKEN";
+
+const issueSessionToken = async (
+  userId: string,
+  period = 10 * 60 * 1000
```

Comment:
minor: I think period is not the best choice of parameter name in this case because period concerns a cyclic event. However, in this case, this is used only one time for setting a expiry datetime. Maybe expireFromNow is better name?

Reply:
how about `expiryDuration`?